pkg/controller/oneagent/oneagent_controller.go

```diff
@@ -325,11 +325,15 @@ func newPodSpecForCR(instance *dynatracev1alpha1.OneAgent) corev1.PodSpec {
 			ReadinessProbe: &corev1.Probe{
 				Handler: corev1.Handler{
 					Exec: &corev1.ExecAction{
-						Command: []string{"pgrep", "-f", "oneagentwatchdog"},
+						Command: []string{
+							"timeout", "-s", "KILL", "-t", "60", // timeout in case grep fails
```

Comment:
60 seconds is way to high compared to the timeout (1) provided by handler. As there is already a timeout set in the handler I suggest to remove this one.

Reply:
This command might not work on some docker images. e.g. it fails on openshift and ubuntu.

Also, as mentioned on Jira ticket, timeout (1) provided by handler does not "always" work, and leaves the thread hanging.